packages/crayons-core/src/components/data-table/data-table.tsx

```diff
@@ -1158,10 +1205,19 @@ export class DataTable {
                 }
               ></fw-checkbox>
             )}
+            {this.colResize && (
```

Comment:
can you move this to a function/variable which return this JSX element and use it in other places. I see some repetition of this code.

Reply:
fixed